com.unity.perception/Runtime/GroundTruth/Labelers/Resources/BoundingBoxPrefab.prefab

```diff
@@ -0,0 +1,171 @@
+%YAML 1.1
```

Comment:
Regarding the use of the Resources folder in packages, there was an edict (so to speak) that packages should (almost) never use the Resources folder to prevent adding unnecessary data to players. I don't think this is a huge deal for Perception, since a few bytes in these players is not that big of a deal, but it's something to read through and consider.

https://docs.google.com/document/d/1sb3SMQHBQMWvPifgabLK40pvptR01bfJeMX9GxRnw28/edit

Reply:
I think we may need to spin off a task and try to get a smart way to do this. maybe using assetbundles or something like that. we could also look into using the new UI Elements, see if that makes it easier to create UIs via scripts